src/main/resources/db/migration/0250/V280__TimeZones.sql

```diff
@@ -0,0 +1,605 @@
+       ('Europe/Zaporozhye'),
+       ('Europe/Zurich'),
+       ('GB'),
+       ('GB-Eire'),
+       ('GMT'),
+       ('GMT0'),
+       ('Greenwich'),
+       ('Hongkong'),
+       ('Iceland'),
+       ('Indian/Antananarivo'),
+       ('Indian/Chagos'),
+       ('Indian/Christmas'),
+       ('Indian/Cocos'),
+       ('Indian/Comoro'),
+       ('Indian/Kerguelen'),
+       ('Indian/Mahe'),
+       ('Indian/Maldives'),
+       ('Indian/Mauritius'),
+       ('Indian/Mayotte'),
+       ('Indian/Reunion'),
+       ('Iran'),
+       ('Israel'),
+       ('Jamaica'),
+       ('Japan'),
+       ('Kwajalein'),
+       ('Libya'),
+       ('MET'),
+       ('Mexico/BajaNorte'),
+       ('Mexico/BajaSur'),
+       ('Mexico/General'),
+       ('MST7MDT'),
+       ('Navajo'),
+       ('NZ'),
+       ('NZ-CHAT'),
+       ('Pacific/Apia'),
+       ('Pacific/Auckland'),
+       ('Pacific/Bougainville'),
+       ('Pacific/Chatham'),
+       ('Pacific/Chuuk'),
+       ('Pacific/Easter'),
+       ('Pacific/Efate'),
+       ('Pacific/Enderbury'),
+       ('Pacific/Fakaofo'),
+       ('Pacific/Fiji'),
+       ('Pacific/Funafuti'),
+       ('Pacific/Galapagos'),
+       ('Pacific/Gambier'),
+       ('Pacific/Guadalcanal'),
+       ('Pacific/Guam'),
+       ('Pacific/Honolulu'),
+       ('Pacific/Johnston'),
+       ('Pacific/Kanton'),
+       ('Pacific/Kiritimati'),
+       ('Pacific/Kosrae'),
+       ('Pacific/Kwajalein'),
+       ('Pacific/Majuro'),
+       ('Pacific/Marquesas'),
+       ('Pacific/Midway'),
+       ('Pacific/Nauru'),
+       ('Pacific/Niue'),
+       ('Pacific/Norfolk'),
+       ('Pacific/Noumea'),
+       ('Pacific/Pago_Pago'),
+       ('Pacific/Palau'),
+       ('Pacific/Pitcairn'),
+       ('Pacific/Pohnpei'),
+       ('Pacific/Ponape'),
+       ('Pacific/Port_Moresby'),
+       ('Pacific/Rarotonga'),
+       ('Pacific/Saipan'),
+       ('Pacific/Samoa'),
+       ('Pacific/Tahiti'),
+       ('Pacific/Tarawa'),
+       ('Pacific/Tongatapu'),
+       ('Pacific/Truk'),
+       ('Pacific/Wake'),
+       ('Pacific/Wallis'),
+       ('Pacific/Yap'),
+       ('Poland'),
+       ('Portugal'),
+       ('PRC'),
+       ('PST8PDT'),
+       ('ROK'),
+       ('Singapore'),
+       ('SystemV/AST4'),
+       ('SystemV/AST4ADT'),
+       ('SystemV/CST6'),
+       ('SystemV/CST6CDT'),
+       ('SystemV/EST5'),
+       ('SystemV/EST5EDT'),
+       ('SystemV/HST10'),
+       ('SystemV/MST7'),
+       ('SystemV/MST7MDT'),
+       ('SystemV/PST8'),
+       ('SystemV/PST8PDT'),
+       ('SystemV/YST9'),
+       ('SystemV/YST9YDT'),
+       ('Turkey'),
+       ('UCT'),
+       ('Universal'),
+       ('US/Alaska'),
+       ('US/Aleutian'),
+       ('US/Arizona'),
+       ('US/Central'),
+       ('US/East-Indiana'),
+       ('US/Eastern'),
+       ('US/Hawaii'),
+       ('US/Indiana-Starke'),
+       ('US/Michigan'),
+       ('US/Mountain'),
+       ('US/Pacific'),
+       ('US/Samoa'),
+       ('UTC'),
+       ('W-SU'),
+       ('WET'),
+       ('Z'),
```

Comment:
This is the only timezone that I had to add manually, because it is expected [here](https://github.com/terraware/terraware-server/blob/main/src/test/kotlin/com/terraformation/backend/tracking/db/plantingSiteStore/PlantingSiteStoreUpdateSeasonsTest.kt#L264) but doesn't seem to exist within the zone IDs.